lib/generators/sufia/install_generator.rb

```diff
@@ -145,5 +145,9 @@ def install_blacklight_gallery
     def datatables
       generate 'jquery:datatables:install bootstrap3'
     end
+
+    def create_workflow
+      template('workflow.json.erb', "config/workflows/default_workflow.json")
```

Comment:
Shouldn't this be something other than default?  We already have a default created by CurationConcerns.  Should this be `meditated_deposit_workflow.json`?